### PR TITLE
Add thumbnails to the file list

### PR DESCRIPTION
**Pull request by mjacobus**
The packed version has an issue:

```
Uncaught Exception:
Error: Cannot find ffmpeg
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/processor.js:136:22
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/capabilities.js:123:9
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/capabilities.js:116:11
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/utils.js:223:16
at F (/Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/node_modules/which/which.js:68:16)
at E (/Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/node_modules/which/which.js:80:29)
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/node_modules/which/which.js:89:16
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/isexe/index.js:42:5
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/isexe/mode.js:8:5
at FSReqCallback.oncomplete (node:fs:198:21)

```

Or as in https://github.com/mjacobus/jw-play/pull/17/commits/5b9e0da5786b233ae0593e3d230bd361fe8fc9d9

````
Uncaught Exception:
Error: spawn ENOTDIR
at ChildProcess.spawn (node:internal/child_process:412:11)
at spawn (node:child_process:707:9)
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/processor.js:152:24
at FfmpegCommand.proto._getFfmpegPath (/Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/capabilities.js:90:14)
at FfmpegCommand.proto._spawnFfmpeg (/Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/processor.js:132:10)
at FfmpegCommand.proto.availableFormats.proto.getAvailableFormats (/Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/capabilities.js:517:10)
at /Applications/JW Play.app/Contents/Resources/app.asar/node_modules/fluent-ffmpeg/lib/capabilities.js:568:14
at fn (/Applications/JW Play.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:638:34)
at Immediate.<anonymous> (/Applications/JW Play.app/Contents/Resources/app.asar/node_modules/async/lib/async.js:554:34)
at processImmediate (node:internal/timers:464:21)
```

- [ ] Fix ffmpeg packing issue
- [ ] Make sure old thumbnails are deleted

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -63,9 +67,35 @@ const createFilePayload = (filePath) => {
...
+    console.log(file);
+  }
+
   return file;
 };
 
+const createVideoThumbnail = (file) => {
+  try {
+    const size = "320x180";
+    const folder = path.join(app.getPath("appData"), "JWPlay", "thumbnails");
```
Comment (JoelMon) [inline src/utils.js:83]: Do you want to save the thumbnails in `jw-play` or `JWplay`, or `JW Play`? There's three directories being used.

Comment (mjacobus) [reply src/utils.js:83]: Hum, cannot remember 🤔 

 It was working in dev mode but  but I hit a wall trying to pack ffmpeg and never came back to this PR.

Comment (mjacobus) [reply src/utils.js:83]: In a mac, this is an example of a thumbnail file:

```
file:///tmp/JWPlay/thumbnails/0aec9c3b-2ed8-4155-9168-6fa02e8d0dfa-320x180.png
```

Comment (mjacobus) [reply src/utils.js]: I renamed the directory to "JW Play" so it uses the same appData directory that was already created.

**Pull request merged**